services/QuillLMS/app/models/question.rb

```diff
@@ -163,6 +165,17 @@ def connect_sentence_combining?
     question_type == TYPE_CONNECT_SENTENCE_COMBINING
   end
 
+  # Translatable
+  private def custom_prompt
```

Comment:
This field also feels like a class variable in that it doesn't appear to change across instances, but it's also configuration so maybe create a general config file and pull in the prompt examples with it?

You could create a new folder:  translatable_configs/{activities, questions, ...}.json

where the file would have two top-level keys:

examples: {
...
}
custom_prompt: {
... 
}

Reply:
ok, I like that!